### PR TITLE
Fixes in Mingle Plugin Meta-data

### PR DESCRIPTION
**Pull request by lmontopo**
Some small fixes to make this consistent with alm v1's copy /
scope of fields.

### CODE REVIEW

File: sdetools/modules/sync_mingle/mingle_metadata.py [hunk 1]
```diff
@@ -25,28 +25,31 @@ def supports_references(self):
     def __init__(self, phases):
         super(MingleMetadata, self).__init__(phases)
 
-        # update server field
-        self.alm_server.default = ''
```
Comment (sibartlett) [inline sdetools/modules/sync_mingle/mingle_metadata.py]: We should have the following placeholder: `your-company.mingle-api.thoughtworks.com`

Comment (lmontopo) [inline sdetools/modules/sync_mingle/mingle_metadata.py]: done :)

File: sdetools/modules/sync_mingle/mingle_metadata.py [hunk 2]
```diff
@@ -25,28 +25,32 @@ def supports_references(self):
...
+        # update placeholder of alm_server
```
Comment (sibartlett) [inline sdetools/modules/sync_mingle/mingle_metadata.py]: Do we need these comments? I feel like the code already describes itself.

Comment (lmontopo) [inline sdetools/modules/sync_mingle/mingle_metadata.py]: I think you're right actually. gonna remove em.

**Comment by sibartlett** [comment]
Good stuff!

**Pull request merged**